src/modality-specific-files/motion.md

```diff
@@ -11,7 +11,7 @@ and can be used as helpful guidance when curating new datasets.
 {{ MACROS___make_filename_template(
 "raw",
 datatypes=["motion"],
-suffixes=["motion", "channels", "events"])
+suffixes=["motion", "events", "physio", "stim"])
```

Comment:
This is not enough to display the physio / stim things in the templates.

See here:

https://bids-specification.readthedocs.io/en/latest/modality-specific-files/motion.html#motion-recording-data

You also need to update the schema:
probably somewhere here: 

https://github.com/bids-standard/bids-specification/blob/098b88e44f4fa1f2024c9be2a6998b2c6b3c9d67/src/schema/rules/files/raw/task.yaml#L54

with something like this I think

```
timeseries__mri:
  $ref: rules.files.raw.task.timeseries
  datatypes:
    - motion
  entities:
    $ref: rules.files.raw.task.timeseries.entities
    tracksys: required
```

Reply:
thanks! I think https://github.com/bids-standard/bids-specification/pull/1458/commits/e212cb1282672ad19eb1e93051bdb7a73fd130dc did the trick